app/src/main/java/com/keylesspalace/tusky/components/notifications/NotificationsRepository.kt

```diff
@@ -31,7 +32,8 @@ import retrofit2.Response
 import javax.inject.Inject
 
 class NotificationsRepository @Inject constructor(
-    private val mastodonApi: MastodonApi
+    private val mastodonApi: MastodonApi,
+    private val gson: Gson
```

Comment:
I'm not completely sure about this. 

This is injected here only so it can be passed down to `NotificationsPagingSource` as a parameter, so it can be provided in the tests as well.

I imagine there's a cleaner way to do this. Thoughts?

Reply:
I think it is fine